webapp/channels/src/components/widgets/menu/menu_group.tsx

```diff
@@ -13,27 +13,23 @@ type Props = {
 /**
  * @deprecated Use the "webapp/channels/src/components/menu" instead.
  */
-export default class MenuGroup extends React.PureComponent<Props> {
-    handleDividerClick = (e: React.MouseEvent): void => {
+const MenuGroup = ({ divider, children }: Props) => {
+    const handleDividerClick = (e: React.MouseEvent): void => {
```

Comment:
`handleDividerClick` does not depend on any prop or state of the component, so it can be defined out of the component.

Why this is good? That way, we only instantiate the function once when we require the file, not every time a component is rendered, leaving less garbage to the GC to collect.